release-notes/opensearch-dashboards-maps.release-notes-2.16.0.0.md

```diff
@@ -0,0 +1,9 @@
+## Version 2.16.0.0 Release Notes
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.16.0
+
+### Features
+* Add support new navigation for maps ([#635](https://github.com/opensearch-project/dashboards-maps/pull/635))
+* 
+### Bug Fixes
+* Fixed broken wms custom layer update ([#601](https://github.com/opensearch-project/dashboards-maps/pull/631))
+* Add data source reference id in data layer search request ([#623](https://github.com/opensearch-project/dashboards-maps/pull/623))
```

Comment:
Looks like the change logis incorrect for #623, it was release in 2.15, please help remove it here. Thanks!